monkey/monkey_island/cc/services/config_schema/definitions/exploiter_classes.py

```diff
@@ -125,6 +125,15 @@
             "info": "Exploits a malicious backdoor that was added to the VSFTPD download archive. "
                     "Logic based on Metasploit module.",
             "link": "https://www.guardicore.com/infectionmonkey/docs/reference/exploiters/vsftpd/"
+        },
+        {
+            "type": "string",
+            "enum": [
+                "DrupalExploiter"
+            ],
+            "title": "Drupal Exploiter",
+            "info": "Exploits a remote command execution vulnerability",
+            "link": ""
```

Comment:
We mustn't forget to update this (add a page + add the link to that page + improve the `info` field).

Reply:
Done 7288fb9814ec962e9ef320d201c2d82c58c2fc27